clients/ordering/src/index.tsx

```diff
@@ -7,6 +7,7 @@ import {
 
 import './index.css';
 import App from './App';
+import { BrowserRouter } from 'react-router-dom';
 import * as serviceWorker from './serviceWorker';
 
 const theme = createMuiTheme({
```

Comment:
Can you merge master into this branch and add `import {theme} from '@jeeves/common';`?

Reply:
You can also remove the `createMuiTheme` usage because we'll be using the `theme` found in common

Reply:
Ok, I'm not super familiar with github...whats the easiest way to merge master into the branch? And to make these changes, do I need to make another pull request after making the changes?

Reply:
- Make sure your local master has the latest from the remote by doing `git pull` from master. 
- Checkout your branch and `git merge master`
- You might have some merge conflicts that you might need to resolve but VSCode does a good job of make it easy to resolve them.
- You shouldn't need to make another PR, just commit the changes and the PR will update

Reply:
This should be all fixed now